### ISSUE TITLE
[Launch API Docs] Implement latest design (look&feel)

### ISSUE THREAD

**Issue by kptdobe** [adobe/Adobe-Developer-Site#10]
Goal
The Helix rendered site must look similar to the current production version (https://docs.adobelaunch.com/).
The helpx and project-helix.io implement the old look&feel, it needs to be updated.

The current production version is rendered with [GitBook](https://www.gitbook.com/). Re-using the production css will be hard because all the css classes have weird names. But there might be a way to find the sources in one of the public repos and reduce the amount of work to re-implement the existing styles. See https://www.gitbook.com/ or https://github.com/GitbookIO/styleguide or https://github.com/GitbookIO

**Comment by kptdobe** [comment]
I looked at the various GitBook repos and could not find something we could reuse easily thus I started to implement the css myself. While having a basic similar look&feel is straight forward, I am not sure if re-building the GitBook styles but also features is really a good idea. And I am pretty sure at one point, we'll want to have our own homemade design even for the doc pages.

My proposal is to replicate the minimum from GitBook to have a start point (something decent to be used) and so that the Launch site looks "similar". Then we will iterate the design with something homemade. @simonwex WDYT ?
